ontology/colao.md

```diff
@@ -4,7 +4,7 @@ id: colao
 title: "Coleoptera Anatomy Ontology (COLAO)"
 contact:
   email: entiminae@gmail.com
-  label: Jennifer C. Giron
+  label: Jennifer C. Girón
```

Comment:
Better not make that change before https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1695 is solved.

Reply:
then should I change the other one in AISM to not have the accent?

Reply:
Oh that appears to be the solution referenced in the issue. Done in 7773c80